CHANGELOG.md

```diff
@@ -0,0 +1,78 @@
+# Change Log
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/)
+and this project adheres to [Semantic Versioning](http://semver.org/).
+
+## [Unreleased]
+
+## [1.1.1] - 30-Apr-2022
+
+* Ruby 3.0 support
```

Comment:
At least on this new one can you add the PR reference as we discussed in the other PR.  Also in the commit message we only need the entries for 1.1.1.